examples/jsm/controls/OrbitControls.js

```diff
@@ -223,8 +246,8 @@ class OrbitControls extends EventDispatcher {
 				spherical.phi = Math.max( scope.minPolarAngle, Math.min( scope.maxPolarAngle, spherical.phi ) );
 
 				spherical.makeSafe();
-
-
+// ???
```

Comment:
Do you mind clarify why you have added `???` here? Maybe a textual description makes it easier to understand temporary solutions.

Reply:
... just another temporary note to me... I will be removing that...